integrated_channels/xapi/serializers.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+
+"""
+Serializers for xAPI data.
+"""
+from __future__ import absolute_import, unicode_literals
```

Comment:
do we need this file?

Reply:
It is not needed here, I will remove it.